sig-instrumentation/annual-report-2023.md

```diff
@@ -20,14 +27,20 @@
       Please raise an issue in kubernetes/community, if the KEP metadata is correct but the generated list is incorrect.
 -->
 
+- [kubernetes-sigs/custom-metrics-apiserver](https://github.com/kubernetes-sigs/custom-metrics-apiserver/blob/master/OWNERS)
+- [kubernetes-sigs/metrics-server](https://github.com/kubernetes-sigs/metrics-server/blob/master/OWNERS)
+- [kubernetes-sigs/prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter/blob/master/OWNERS_ALIASES)
+
 3. Did you have community-wide updates in 2023 (e.g. KubeCon talks)?
 
 <!--
   Examples include links to email, slides, or recordings.
 -->
 
-4. KEP work in 2023 (v1.27, v1.28, v1.29):
+- [SIG Instrumentation Introduction and Deep Dive (KubeCon NA '23)](https://youtu.be/Lf5h8bPrSBM?si=HOv63HqVRIAB2mjw)
+- [SIG Instrumentation Introduction and Deep Dive (KubeCon EU '24)](https://youtu.be/Sx1jmIJhfyA?si=Mita7yfvSCcllzta)
```

Comment:
This is the annual report for 2023. I think you can take out the KubeCon EU '24 entry here and add in the next report.